sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -0,0 +1,458 @@
+"""A PipelineRunner using the SDK harness.
+"""
+import collections
+import json
+import logging
+import Queue as queue
```

Comment:
For consistency. The name of the class is `Queue`. (This wart was fixed in Python 3.)